src/electionguard/publish.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+from jsons import set_serializer
+from os import mkdir, path
+from jsons import set_serializer, dump
+
+
+from .ballot_box import CiphertextBallot
+from .decryption_mediator import PlaintextTally
+from .election import CiphertextElectionContext, ElectionConstants, ElectionDescription
+from .key_ceremony import CoefficientValidationSet
+from .tally import CiphertextTally
+from .group import ElementModP, ElementModQ
+
+RECORD_DIR = "output" + path.sep
+COEFFICIENTS_DIR = RECORD_DIR + "coefficients" + path.sep
+BALLOTS_DIR = RECORD_DIR + "encrypted_ballots" + path.sep
+SPOILED_DIR = RECORD_DIR + "spoiled_ballots" + path.sep
+
+DESCRIPTION_FILE_NAME = "description"
+CONTEXT_FILE_NAME = "context"
+CONSTANTS_FILE_NAME = "constants"
+ENCRYPTED_TALLY_FILE_NAME = "encrypted_tally"
+TALLY_FILE_NAME = "tally"
+
+COEFFICIENT_PREFIX = "coefficient_validation_set_"
+BALLOT_PREFIX = "ballot_"
+
+
+def publish(
+    description: ElectionDescription,
+    context: CiphertextElectionContext,
+    constants: ElectionConstants,
+    coefficient_validation_sets: List[CoefficientValidationSet],
```

Comment:
In some cases, there will be no coefficients hanging around because it's just a single trustee. Should this list be empty in those cases? `Optional`?

Reply:
We can make the parameter optional by defaulting to empty list.